### ISSUE TITLE
How to get 0x55 in pgen_spec?

### ISSUE THREAD

**Issue by biona001** [chrchang/plink-ng#223]
Hello,

In the example at end of section 2.3.5, I wonder how can I get `0x55`?

Also, can you clarify by "every other even sample..." do you mean `0001 0001 0001....` or `0100 0100 0100 0100....`?
+ In the first case I get `0x88`
+ The second case I get `0x44`

**Comment by chrchang** [comment]
By "every other even sample", I meant all even sample IDs except for ID 10000.  I'll reword this to eliminate the ambiguity.

**Comment by biona001** [comment]
Oh, I see. Thank you very much.
